### DISCUSSION TITLE
Broken mesh from xda files

### DISCUSSION THREAD

**Discussion by favinom** [idaholab/moose#24700] [Q&A General] [answered]
Hi all,

I am trying to do the following:

1) call BreakMeshByBlockGenerator on a mesh
2) save the mesh in an xda file
3) read the mesh from the xda file in another script

The problem is that the mesh read from xda does not "see" the interfaces.
I think this is due to the fact that the neighboring map has been lost in the xda file.

Is there a way to read a broken mesh with the correct interfaces?

Thanks
Marco

**Comment by GiudGiud** [comment, marked as answer]
Hello

xda might be losing some information
Let's check by trying with another mesh format type : exodus for example

Guillaume
